platform/ios/src/MGLMapView.mm

```diff
@@ -2962,6 +2973,23 @@ - (MGLAnnotationImage *)defaultAnnotationImage
     return annotationImage;
 }
 
+- (MGLAnnotationImage *)invisibleAnnotationImage
+{
+    MGLAnnotationImage *annotationImage = [self dequeueReusableAnnotationImageWithIdentifier:MGLInvisibleStyleMarkerSymbolName];
+    
+    if (!annotationImage)
+    {
+        UIGraphicsBeginImageContext(CGSizeMake(1, 1));
```

Comment:
Is this creating an image with width/height 1/1 and using this in core as image for the pointannotation? As far as I can see this is the same hack I'm using on Android: https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/MarkerView.java#L299